scripts/eslint-plugin-fig-linter/rules/no-empty-array-values.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  meta: {
+    type: "problem",
+    fixable: "code",
+  },
+  create: function (context) {
+    return {
+      Property(node) {
+        if (
+          node.key.name === "options" ||
+          node.key.name === "subcommands" ||
+          node.key.name === "args"
+        ) {
```

Comment:
Can be simplified to the above if you want:
```suggestion
        if (["options", "subcommands", "args"].includes(node.key.name)) {
```